ppml/kms-client/README.md

```diff
@@ -46,14 +46,16 @@ Then, you are allowed to generate primary and data keys, and use them to encrypt
 ```bash
 export INPUT_DIR_PATH=YOUR_DATA_FILE_DIRECTORY_PATH # For example, multiple CSV files are in this directory
 export KMS_SERVER_IP=YOUR_KMS_SERVER_IP # IP address of node where the previous KMS server is deployed
+export APP_ID=YOUR_APP_ID
+export APP_KEY=YOUR_APP_KEY
 
 # Step 1. Generate Primary Key And Data Key
 # Keys in ciphertext are saved at current path as default
-python ./KMS_Client.py --api generate_primary_key --ip $KMS_SERVER_IP
-python ./KMS_Client.py --api generate_data_key --ip $KMS_SERVER_IP --pkp ./encrypted_primary_key
+python ./KMS_Client.py --api generate_primary_key --ip $KMS_SERVER_IP --appid $APP_ID --appkey $APP_KEY
+python ./KMS_Client.py --api generate_data_key --ip $KMS_SERVER_IP --pkp ./encrypted_primary_key --appid $APP_ID --appkey $APP_KEY
 

```

Comment:
@Uxito-Ada I removed --appid and --apikey option, and use environment only. Please update documents.

Reply:
resolved